week-1/G-strings-methods/exercise.js

```diff
@@ -1,3 +1,6 @@
 // Start by creating a variable `message`
 
-console.log(message);
+var name = "ahmed";
+var nameLength = name.length;
+
+console.log("my name is " + name + " and my name is 5 characters long");
```

Comment:
Hey Ahmed, I think you can use the variable _nameLength_ in the console in order to make nameLength dynamic. :smile:

Reply:
Thanks ,ivinapontes